.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,112 @@
+name: Prepare release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      generate_changelog:
+        required: false
+        default: true
+        type: boolean
+      update_db:
+        required: false
+        default: false
+        type: boolean
+      db_dump_path:
+        required: false
+        type: string
+      migration_command:
+        required: false
+        default: './vendor/bin/phinx migrate'
+        type: string
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Update package version
+        run: |
+          if [ -f "package.json" ]; then
+            npm pkg set version=${{ inputs.version }}
+          fi
+
+      - name: Update changelog file
+        if: ${{ inputs.generate_changelog == true }}
+        uses: uniquesca/continuous-integration/update-changelog@main
+        with:
+          target_version: ${{ inputs.version }}
+
+      - name: Update DB dump
+        if: ${{ inputs.update_db == true }}
+        run: |
+          php composer.phar config --auth github-oauth.github.com ${{ secrets.ACCESS_TOKEN }}
```

Comment:
All secrets in the reusable workflow have to be defined along with the inputs. This step fails now as it can't access the secret.